### DISCUSSION TITLE
Only open specific sidebar sections by default?

### DISCUSSION THREAD

**Discussion by anthonyshew** [fuma-nama/fumadocs#262] [Ideas]
Another feature request!

I'd like to be able to have only certain sections of the sidebar open by default, as shown in this screenshot.

![CleanShot 2024-02-20 at 23 13 00](https://github.com/fuma-nama/fumadocs/assets/35677084/889f568b-b493-4b5a-87c8-610553bd9caf)

I'm using `defaultOpenLevel: 0` to keep everything closed at the start - but would love to have the first two sections open when users first visit the docs.

Is this possible?

**Comment by fuma-nama** [comment]
It is not possible for now, welcome to open a feature request in Issues

**Reply by anthonyshew** [reply]
Opened #265. Sorry again for making these Discussions! 😄
